src/screens/Wallets/Receive/ReceiveConnect.tsx

```diff
@@ -26,15 +32,63 @@ const ReceiveConnect = ({
 	const { t } = useTranslation('wallet');
 	const { isSmallScreen } = useScreenSize();
 	const lightningBalance = useLightningBalance(true);
-	const { amount, jitOrder } = useSelector(receiveSelector);
+	const [feeEstimate, setFeeEstimate] = useState(0);
+	const [isLoading, setIsLoading] = useState(false);
+	const { amount } = useSelector(receiveSelector);
+	const invoice = useSelector(receiveSelector);
+	const blocktank = useSelector(blocktankInfoSelector);
 
-	const order = jitOrder!;
-	const payAmount = amount - order.feeSat;
-	const displayFee = useDisplayValues(order.feeSat);
+	const { maxChannelSizeSat } = blocktank.options;
 
-	const onContinue = (): void => {
+	const requestData: ICreateOrderRequest = {
+		lspBalanceSat: amount,
+		channelExpiryWeeks: DEFAULT_CHANNEL_DURATION,
+		options: {},
+	};
+
+	async function feeEstimation(): Promise<void> {
+		try {
+			const estimate = await estimateOrderFee(requestData);
+			if (estimate.isOk()) {
+				setFeeEstimate(estimate.value);
+			} else {
+				console.error('Error in estimate');
+			}
+		} catch (error) {
+			console.error('Error', error);
+		}
+	}
```

Comment:
Should wrap this in `useEffect` so it doesn't run on every render:

```suggestion
	useEffect(() => {
		const getFeeEstimation = async (): Promise<void> => {
			const estimate = await estimateOrderFee({
				lspBalanceSat: amount,
				channelExpiryWeeks: DEFAULT_CHANNEL_DURATION,
			});
			if (estimate.isOk()) {
				setFeeEstimate(estimate.value);
			}
		};

		getFeeEstimation();
	}, [amount]);
```

Reply:
To avoid the type error update in `types/blocktank.ts`:

```
export interface ICreateOrderRequest {
	lspBalanceSat: number;
	channelExpiryWeeks: number;
	options?: Partial<ICreateOrderOptions>;
}
```